firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -233,6 +239,21 @@ public void deleteFieldIndex(FieldIndex index) {
     }
   }
 
+  @Override
+  public void createTargetIndices(Target target) {
+    hardAssert(started, "IndexManager not started");
+
+    if (!autoClientIndexingEnabled) return;
```

Comment:
I am not sure this is useful..nobody would call createTargetIndices if this is configured false?